src/hooks/useKeyboardShortcut/keyCodes.ts

```diff
@@ -0,0 +1,210 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+export const modifierKeys = {
+  AltLeft: 'AltLeft',
+  AltRight: 'AltRight',
+  ControlLeft: 'ControlLeft',
+  ControlRight: 'ControlRight',
+  OSLeft: 'OSLeft',
+  OSRight: 'OSRight',
+  ShiftLeft: 'ShiftLeft',
+  ShiftRight: 'ShiftRight',
+  MetaLeft: 'MetaLeft',
+  MetaRight: 'MetaRight',
+};
+
+export type Modifier = keyof typeof modifierKeys;
+
+/**
+ * Keyboard event code values based on MDN documentation
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_code_values
+ */
+export const keyCodes = {
```

Comment:
This is quite a bit of code to add, could we make this a const enum instead? Not sure if that works with a build package 🤔

Reply:
This const is not used by the hook itself, so will only become part of the app bundle when the user explicitly imports it.

Internally it's only used for typing. And exported as a convenience in case the user likes to make use of it.